era5cli/fetch.py

```diff
@@ -157,6 +166,10 @@ def _define_outputfilename(self, var, years):
                   else self.outputprefix)
         yearblock = f"{start}-{end}" if not start == end else f"{start}"
         fname = f"{prefix}_{var}_{yearblock}_{self.period}"
+        if self.area:
+            directions = ['N', 'W', 'S', 'E']
+            coords = [str(c) for c in self._round_area()]
```

Comment:
I don't think there's an ideal solution here. Maybe convert to int instead of rounding to 2 decimals to avoid decimal points in the filenames? I think that'll be fine for most use cases, and much more friendly to the eye

Reply:
There's a slight possibility for overwriting files or not being able to distinguish very specific areas, but I'm happy to apply this.